Shared/Samples/Style symbols from mobile style file/StyleSymbolsFromMobileStyleFileView.SymbolOptionsListView.swift

```diff
@@ -0,0 +1,121 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension StyleSymbolsFromMobileStyleFileView {
+    struct SymbolOptionsListView: View {
+        /// The display scale of the environment.
+        @Environment(\.displayScale) private var displayScale
+        
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        /// The list of all the symbols and their associated data from the symbol style.
+        @State private var symbols: [SymbolDetails] = []
+        
+        var body: some View {
+            VStack {
+                Image(uiImage: model.currentSymbol?.image ?? UIImage())
+                
+                List {
+                    // Create a picker for each symbol category.
+                    ForEach(SymbolCategory.allCases, id: \.self) { category in
+                        Section(category.label) {
+                            Picker("Symbol Names", selection: $model.symbolOptionSelections.categoryKeys[category]) {
+                                ForEach(symbols.filter { $0.category == category.rawValue }, id: \.key) { symbol in
+                                    Label {
+                                        Text(symbol.label)
+                                    } icon: {
+                                        Image(uiImage: symbol.image)
+                                    }
+                                    .tag(symbol.key as String?)
+                                }
+                            }
+                            .pickerStyle(.inline)
+                            .labelsHidden()
+                        }
+                    }
+                    
+                    Section {
+                        ColorPicker("Color", selection: $model.symbolOptionSelections.color)
+                        VStack {
+                            HStack {
+                                Text("Size")
+                                Spacer()
+                                Text(model.symbolOptionSelections.size.formatted(.number.rounded()))
+                                    .foregroundColor(.secondary)
+                            }
+                            Slider(value: $model.symbolOptionSelections.size, in: 20...60, step: 1)
+                        }
+                    }
+                }
+            }
+            .task(id: model.symbolOptionSelections) {
+                // Update the current symbol when an option selection changes.
+                await model.updateCurrentSymbol(displayScale: displayScale)
+            }
+            .task(id: displayScale) {
+                // Update the symbols when the display scale changes.
+                symbols = await getSymbols(displayScale: displayScale)
```

Comment:
when does the display scale ever change?

Reply:
The display scale changes when the app is moved to another screen which has a different resolution, which really can only happen on Mac Catalyst and even then, it is unlikely. I have been including it in the style samples since Ting said it was better for accessibility, and I found the pattern in the [Show utility associations](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Samples/Show%20utility%20associations/ShowUtilityAssociationsView.swift#L75-L78) and [Display content of utility network container](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Samples/Display%20content%20of%20utility%20network%20container/DisplayContentOfUtilityNetworkContainerView.swift#L70-L73) samples. Let me know if you want me to remove as it won't make much of a difference.

Reply:
Good info, thank you! I would leave it then.